templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        if i&gt;1:
</pre><p>Most of the code in this notebook should be extracted into individual helper functions that are imported and called</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>